docs/about/release-notes.rst

```diff
@@ -43,12 +43,15 @@ Features
 
 * Added alignment flag to ``Variable`` `#3144 <https://github.com/scipp/scipp/pull/3144>`_.
 * Handle alignment of coordinates in operations `#3153 <https://github.com/scipp/scipp/pull/3153>`_.
+* ``sc.compat.from_pandas`` now has additional arguments `#3169 <https://github.com/scipp/scipp/pull/3169>`_.
+* Added ``sc.io.load_csv`` `#3169 <https://github.com/scipp/scipp/pull/3169>`_.
 
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
 * Coordinates are no longer converted to attributes when slicing or by ``transform_coords``.
   Instead, they become unaligned `#3153 <https://github.com/scipp/scipp/pull/3153>`_.
+* By default, the row coordinate produced by ``sc.compat.from_pandas`` now has ``unit=None`` (used to be dimensionless) `#3169 <https://github.com/scipp/scipp/pull/3169>`_.
```

Comment:
This is referring to the coord made from the Pandas index? What if the index is not an integer, but something else (like floats)?

Reply:
Pandas lets you convert any column to an index. Scipp can have physical quantities as coordinates, so why wouldn't you have the same with Pandas indices?

Reply:
What is your thinking for giving that coordinate a `None` unit? Is it because it's something we are adding by hand that was not in the original pandas data frame?
So the logic is now that everything from the data frame is given `dimensionless` if no unit is found in the header, and anything we add by hand after is given unit `None`?

I am now wondering why we have the `row` coordinate in the first place? The data array has dimension `row`, but does it really need the coord as well? (there's probably a good reason, but I forget what that is right now).

Reply:
Don't you always apply some reduction operation after `groupby` so that the index is lost anyway? I tried looking at the `groupby` examples on Pandas but couldn't find one where the index still seemed important after the `groupby` operation.

Reply:
The index is not lost after the reduction: The new index is the grouping key.

Reply:
ok